setup.py

```diff
@@ -18,6 +18,7 @@
           'colorama',
           'argparse',
           'configparser',
+          'BeautifulSoup',
```

Comment:
Weird, because BeautifulSoup is also imported for Python3. Maybe it's not supposed to be?